spec/acceptance/suites/default/redis_sentinel_one_node_spec.rb

```diff
@@ -9,6 +9,13 @@
                  'redis'
                end
 
+  # We're testing with `manage_repo` true.  In redis-sentinel 5, the daemon has its own rundir
+  pidfile = if fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemmajrelease') == '16.04'
```

Comment:
Could you implement this in pure puppet below?

Reply:
I suppose I may as well.  Ideally, the module itself would set the default correctly, but that's for another day.

Reply:
Done (assuming I've not messed it up).

Reply:
I messed it up...